code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -15,6 +15,20 @@
 	var/stealth = FALSE
 	var/can_sneak_attack = FALSE
 	var/stealth_alpha_multiplier = 1
+	var/stealth_duration = -1
+	var/stealth_timer
+	var/disable_on_signals = list(
+		COMSIG_XENOMORPH_GRAB,
+		COMSIG_XENOMORPH_THROW_HIT,
+		COMSIG_LIVING_IGNITED,
+		COMSIG_LIVING_ADD_VENTCRAWL,
+		COMSIG_XENOABILITY_MIRAGE_SWAP
+	)
+	var/disable_on_obj_slash = TRUE
+	var/disable_on_mob_slash = TRUE
+	var/sneak_attack_armor_pen = HUNTER_SNEAK_SLASH_ARMOR_PEN
+	var/pounce_break_stealth = TRUE
+	var/pounce_hit_break_stealth = TRUE
```

Comment:
Doc? Why aren't all those bools flags?

Reply:
Sure I'll add docs, bool flags.. Assuming what i think that is, thought its not necessary since it's just true or false nothing else for pretty much everything.